### DISCUSSION TITLE
xview results reproduction

### DISCUSSION THREAD

**Discussion by KaizerJ** [fcakyon/small-object-detection-benchmark#19] [Q&A] [answered]
Hello, 

Which set was used for evaluation on xView dataset? Validation dataset labels weren't publish, were they? If so, how can I get them to reproduce the results?

Thanks in advance

**Comment by fcakyon** [comment, marked as answer]
You can use these script to create the xview splits: https://github.com/fcakyon/small-object-detection-benchmark/blob/main/xview/xview_to_coco.py

**Reply by KaizerJ** [reply]
Thanks for the quick reply!
